wafer/talks/migrations/0001_initial.py

```diff
@@ -23,7 +23,7 @@ class Migration(migrations.Migration):
                 ('status', models.CharField(default=b'P', max_length=1, choices=[(b'A', b'Accepted'), (b'R', b'Not Accepted'), (b'P', b'Under Consideration')])),
                 ('_abstract_rendered', models.TextField(editable=False, blank=True)),
                 ('authors', models.ManyToManyField(related_name='talks', to=settings.AUTH_USER_MODEL)),
-                ('corresponding_author', models.ForeignKey(related_name='contact_talks', to=settings.AUTH_USER_MODEL)),
+                ('corresponding_author', models.ForeignKey(related_name='contact_talks', to=settings.AUTH_USER_MODEL, on_delete=models.CASCADE)),
```

Comment:
Any specific reason to not reformat this long line like the other changes?

Reply:
I think I meant to come back to that. The whole area is a lot messier.